CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Mapbox welcomes participation and contributions from everyone.
 * Update to MapboxCoreMaps 10.4.0-beta.1 and MapboxCommon 21.2.0-beta.1. ([#1126](https://github.com/mapbox/mapbox-maps-ios/pull/1126))
 * Exposed APIs to allow positioning of other views relative to the logoView, compassView, scaleBarView and attributionButton. ([#1130](https://github.com/mapbox/mapbox-maps-ios/pull/1130))
 * Add `GestureOptions.pinchPanEnabled` and `.pinchZoomEnabled`. ([#1092](https://github.com/mapbox/mapbox-maps-ios/pull/1092))
+* Zoom focal point ([#1122](https://github.com/mapbox/mapbox-maps-ios/pull/1122))
+* Add focalPoint property to zoom and rotate gestures ([#1122](https://github.com/mapbox/mapbox-maps-ios/pull/1122))
```

Comment:
nit: duplicated change log entries for the same PR, probably we want to remove the first one?

Reply:
Well spotted, rebase artefacts, removed it.